project/wst24365888/README.md

```diff
@@ -0,0 +1,70 @@
+# Stream VByte
+
+## Basic Information
+
+A `C++` implementation with `Python` bindings of Stream VByte, which is an efficient integer compression technique.
+
+## Problem to Solve
+
+Integer compression can often be used in many places. One of the most famous is VByte, which is everywhere. 
+
+In 2015, a well-known speedup version was proposed, Masked VByte (Vectorized VByte Decoding), which data format is the same as the standard VByte, using SIMD to speed up the decoding process, but its proposed algorithm has some limitations, such as only considering first 12 bytes in 16 byte input data and a hardcoded table that requires a lot of pre-computation. 
+
+Later, one of its authors, Prof. Daniel Lemire, proposed Stream VByte which uses a different data format, splitting the control bytes and data bytes. This results in better access speed and it's more suitable for SIMD approaches.
+
+This project is a `C++` implementation of Stream VByte, including `Python` bindings in newer versions, which was previosly lacking.
+
+## Prospective Users
+
+- Users that want to make integer compression
+  - `MIDI` files
+  - database
+  - IoT
+  - `RPC` frameworks like [thrift](https://github.com/facebook/fbthrift), [grpc](https://github.com/grpc/grpc) (they use `protobuf (varint)` instead)
+
+## System Architecture
+
+![System Architecture](./assets/system-architecture.png)
+
+## API Description
+
+- `Python` API

```

Comment:
Do you plan to support Python byte and str types?

Reply:
If I still have time after finish current `Python` APIs in proposal, I will. :)

Reply:
Fair enough.